seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SeataProperties.java

```diff
@@ -41,6 +39,14 @@
      * transaction service group
      */
     private String txServiceGroup;
+    /**
+     * Whether enable auto proxying of datasource bean
+     */
+    private boolean enableAutoDataSourceProxy = true;
+    /**
+     * Whether use JDK proxy instead of CGLIB proxy
+     */
+    private boolean useJdkProxy = false;
```

Comment:
add them to yml and properties under script

Reply:
OK

Reply:
done